cornac/models/cdl/cdl.py

```diff
@@ -40,12 +40,12 @@ def sdae(X_corrupted, layers, dropout_rate=0.0, act_fn='relu', seed=None, name="
     L = len(layers)
     rng = get_rng(seed)
     weights, biases = [], []
-    with tf.variable_scope(name):
+    with tf.compat.v1.variable_scope(name):
```

Comment:
To avoid verbosity, we only need to update the import statement as
```python
import tensorflow.compat.v1 as tf
```
and leave the rest unchanged.

Reply:
Ok I'll update it, thanks!